cmd/registry/cmd/compute/score/score.go

```diff
@@ -30,7 +30,7 @@ import (
 
 func Command() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "score",
+		Use:   "score RESOURCE",
```

Comment:
Maybe this should be PATTERN, but we can revisit that.

Reply:
No, you're right, PATTERN is correct.